cumulus/client/cirrus-executor/src/aux_schema.rs

```diff
@@ -90,22 +90,23 @@ pub(super) fn write_execution_receipt<Backend: AuxStore, Block: BlockT>(
 }
 
 /// Load the execution receipt associated with a block.
-pub(super) fn load_execution_receipt<Backend, Hash>(
+pub(super) fn load_execution_receipt<Backend, Hash, PBlock>(
```

Comment:
I thought that too, but the compiler actually can't infer the type so I have to specify it explicitly.

Reply:
It should be able to infer if you separate as I suggested into two types. If you make generic on `PBlock` then it can infer block number and hash, but not the rest of the block contents.

Reply:
Oh, get it. I'll update my next PR if CI is green this time, otherwise, I'll update in this PR.

Reply:
Updated in this one :D